README.md

```diff
@@ -1,11 +1,53 @@
-# *A*ndroid *L*ibrary for *P*assport *I*nformation via *N*FC *E*xtraction
+# **A**ndroid **L**ibrary for **P**assport **I**nformation via **N**FC **E**xtraction
 
-## Usage
+
+ALPINE is a library to simplify the extraction of passport information by reading data off of the NFC chip
+
+## Download
```

Comment:
Download or Setup 🤔 Not sure we usually use `Download` for `lib` right?
```suggestion
## Setup
```

Reply:
As of right now the only way to use the library is to download the project since #32 is not yet merged